airflow/cli/commands/role_command.py

```diff
@@ -41,3 +39,259 @@ def roles_create(args):
     appbuilder = cached_app().appbuilder  # pylint: disable=no-member
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
+
+
+mapping = {
+    # ========================= Airflow Views
+    ("Airflow", "can_blocked"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_clear"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_delete")],
```

Comment:
I wonder if this should be `(TI, can_edit)` -- we don't actually delete the TI row when clearing, just clear/null out its status.